lms/static/js/verify_student/views/make_payment_step_view.js

```diff
@@ -46,6 +46,8 @@ var edx = edx || {};
         _getPaymentButtonText: function(processorName) {
             if (processorName.toLowerCase().substr(0, 11)=='cybersource') {
                 return gettext('Pay with Credit Card');
+            } else if (processorName.toLowerCase()=='paypal') {
+                return gettext('Pay with PayPal');
```

Comment:
We should mention to Griff that we've gone ahead with the text 'PayPal' here instead of a logo.